test/system/710-kube.bats

```diff
@@ -135,4 +135,44 @@ status  | =  | null
     run_podman rmi $(pause_image)
 }
 
+@test "podman kube generate - deployment" {
+    local pname=p$(random_string 15)
+    local cname1=c1$(random_string 15)
+    local cname2=c2$(random_string 15)
+
+    run_podman pod create --name $pname
+    run_podman container create --name $cname1 --pod $pname $IMAGE top
+    run_podman container create --name $cname2 --pod $pname $IMAGE bottom
+
+    skip_if_remote "containersconf needs to be set on server side"
```

Comment:
Please move to the first line (under `@test`). To prevent wasteful podman create, and to make it ultra-clear to a reader

Reply:
fixed